### PR TITLE
feat(settings): configure layout responsiveness for settings page

### LINKED ISSUE #381: Responsive Settings Page
Check out how the chat page works - the sidebar is the default mobile view

Here is chat:
When you make it mobile size, it collapses to sidebar:
Screen Shot 2022-11-22 at 2 59 01 PM

You can click a chat to show the content window, and nav back with that arrow hidden on desktop mode:
Screen Shot 2022-11-22 at 2 59 47 PM

Current settings:
Screen Shot 2022-11-22 at 3 00 03 PM

Question - do you think we should show the bottom nav on both settings pages, or make them go back to the sidebar?

### PR DESCRIPTION
**Pull request by nathan-power**
### What this PR does 📖
- Adds responsiveness and functionality to collapse and display sidebar for settings page.
- Adds header for sidebar page with working back button and dynamic title.
- Adds functionality to collapse and display sidebar for files page (with working back button).

### Which issue(s) this PR fixes 🔨
- Resolve #381

### CODE REVIEW

**Comment by phillsatellite** [comment]
When tested on my side I see all the headers in Settings but didnt see a collapse or back button, I had Sara verify and she experienced same issue she also mentioned she did control+a to show any buttons that might be hiding and no luck. 

Captura_de_ecra_2022-11-23_as_18 49 24

**Review by InfamousVague — APPROVED** [review]
Approved pending QA

**Comment by phillsatellite** [comment]
After testing I found a few UI things that I think could just be made into some tickets instead of holding up the PR but just that the Profile page and the Extensions page are a little off center but other than that looked good 👍 
Screenshot 2022-11-23 at 4 23 05 PM
Screenshot 2022-11-23 at 4 14 08 PM

**Review by phillsatellite — APPROVED** [review]

**Comment by nathan-power** [comment]
> After testing I found a few UI things that I think could just be made into some tickets instead of holding up the PR but just that the Profile page and the Extensions page are a little off center but other than that looked good 👍 

Hi Phil 👋
I'm currently working on the responsiveness for the friends page so I'll try adding these fixes along with that PR

**Pull request merged**
